File: Data Structures & Algorithms/top-k-elements-in-list/submission-2.cpp
class Solution {
public:
    vector<int> topKFrequent(vector<int>& nums, int k) {
        vector<int> result;
        map<int, int> my_map;
        for (auto& num : nums) {
            my_map[num]++;
        }
        vector<pair<int, int>> v(my_map.begin(), my_map.end());
        sort(v.begin(), v.end(), [](pair<int, int> & a, pair<int, int> & b) {return a.second > b.second;});
        for (int i = 0; i < k; i++) {
            result.push_back(v[i].first);
        }
        return result;
    }
};

File: Data Structures & Algorithms/kth-largest-integer-in-a-stream/submission-2.py
import heapq
class KthLargest:
    def __init__(self, k: int, nums: List[int]):
        nums.sort()
        self.h = nums
        self.k = k

    def add(self, val: int) -> int:
        self.h.append(val)
        self.h.sort()
        return self.h[len(self.h) - self.k]
